requirements.in

```diff
@@ -1,7 +1,9 @@
--e git://github.com/mbertheau/python-emarsys@master#egg=python-emarsys
+-e git://github.com/mbertheau/python-emarsys@0.3#egg=python-emarsys
+
 # lowest supported django version
-Django<1.8
+Django>1.8
```

Comment:
That seriously decreases the value of the work for us. Django 1.9 is supported until 8 months from now, whereas Django 1.8 is supported 20 months from now.

Additionally the django documentation states here https://docs.djangoproject.com/en/1.8/releases/1.8/ that Python 3.5 is supported. In fact a problem with HTMLParseError is mentioned here https://code.djangoproject.com/ticket/23763 and has been fixed here https://github.com/django/django/commit/b07aa52e8a8e4c7fdc7265f75ce2e7992e657ae9

Is there anything else preventing us from using Django 1.8?